src/lib/components/WebvizPluginPlaceholder.react.js

```diff
@@ -82,18 +82,19 @@ export default class WebvizPluginPlaceholder extends Component {
                             <WebvizToolbarButton
                                 icon={faCameraRetro}
                                 tooltip="Take screenshot"
-                                onClick={() =>
+                                onClick={() => {
+                                    const filename = this.props.screenshot_filename;
                                     html2canvas(
                                         document.getElementById(this.props.id)
                                     ).then(canvas =>
                                         canvas.toBlob(blob =>
                                             download_file(
-                                                "webviz-screenshot.png",
+                                                filename,
```

Comment:
Should we use `this.props.screenshot_filename` directly here, and thereby saving the new line `const filename = this.props.screenshot_filename;` and enclosing `{}`?